src/compiler/compile.ts

```diff
@@ -69,6 +69,19 @@ export function compile(
         trackInfos = getRelativeTrackInfo(specCopy, theme).trackInfos;
     }
 
+    // publish the fixed spec
+    // used for alt-gosling
+    try {
+        if (PubSub) {
+            PubSub.publish('specTraversed', {
+                id: specCopy.id,
+                data: specCopy
```

Comment:
Maybe more intuitive and consistent with how we use "data" and "spec" in other API functions

```suggestion
                spec: specCopy
```

Reply:
Ah I see, I had already written this piece before the [standardization](https://github.com/gosling-lang/gosling.js/commit/1c2b7669660856cae98f88ba38d9326b788c75e1)! Thanks!